hw/kdrive/src/kdrive.h

```diff
@@ -345,6 +357,9 @@ void
 int
  KdProcessArgument(int argc, char **argv, int i);
 
+void
```

Comment:
This function is then called by Xfbdev later, correct ?
Maybe the other kdrive-based servers should also call it, for consistency (even they might not actually implement any of these function vectors) ?

Reply:
yeah, seen it :)